app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from os import makedirs, rename, remove, chdir, getcwd, path
+
+
+def move_file(command: str) -> None:
+    command_name = command.split()[0]
+    source_file_name = command.split()[1]
+    destination_name = command.split()[2]
```

Comment:
Simplify this operation by using unpacking
```suggestion
    command_name, source_file_name, destination_name, *_  = command.split()
```